libs/scripts/delayed.js

```diff
@@ -12,9 +12,14 @@
 
 export const loadJarvisChat = async (getConfig, getMetadata, loadScript, loadStyle) => {
   const config = getConfig();
-  const jarvis = getMetadata('jarvis-chat');
+  const jarvis = getMetadata('jarvis-chat')?.toLowerCase();
   if (!config.jarvis?.id || !config.jarvis?.version) return;
-  if (jarvis === 'on') {
+
+  if (jarvis) {
+    const desktopViewport = window.matchMedia('(min-width: 900px)').matches;
+    if (jarvis === 'mobile' && desktopViewport) return;
```

Comment:
What needs to be configured so it's available for both desktop and mobile? Just any string that is not 'mobile' or 'desktop' ?

Reply:
it just needs a value, we'll document it as `on` - `mobile, desktop, on`